interface/forms/prior_auth/templates/prior_auth/general_new.html

```diff
@@ -34,20 +48,134 @@
 
 -->
 </style>
+<script>
+function showHint(str) {
+    if (str.length == 0) { 
+        document.getElementById("txtHint").innerHTML = "";
+        return;
+    } else {
+        var xmlhttp = new XMLHttpRequest();
+        xmlhttp.onreadystatechange = function() {
+            if (xmlhttp.readyState == 4 && xmlhttp.status == 200) {
+                document.getElementById("txtHint").innerHTML = xmlhttp.responseText;
+            }
+        };
+        xmlhttp.open("GET", "gethint.php?q=" + str, true);
+        xmlhttp.send();
+    }
+}
+
+ function validate() {
+            var EnteredDate = document.getElementById("auth_from").value; // get the auth from date
+
+            var today = document.getElementById("auth_to").value;
+
+            if (EnteredDate > today) {
+                alert("FROM: date is greater than TO: date ");
+				return false;
+            }
+            var AuthReq = document.getElementById("noAuthReq").checked;	
+            var Claims = document.getElementById("claims").value;
+            //var ClearReq = document.getElementById("clear_req").checked; // Not needed will delete
+			var n = Claims.length;
+			
+            if(document.getElementById("noAuthReq").checked && n === 0){
+			    
+			      alert("Please fill in number of claims");
+			      return false;
+				 
+            }
+         
+        }
+
+</script>
+
 {/literal}
 </head>
 <body bgcolor="{$STYLE.BGCOLOR2}">
+
 <p><span class="title">Prior Authorization Form</span></p>
 <form name="prior_auth" method="post" action="{$FORM_ACTION}/interface/forms/prior_auth/save.php">
 <table>
 <tr>
-	<td>Prior Authorization Number</td><td><input type="text" size="35" name="prior_auth_number" value="{$prior_auth->get_prior_auth_number()}"></td>
+    <td align="right">No Auth# Req:</td><td><input type="radio" name="not_req" value="Yes" id="noAuthReq" title="Select when No Authorization Number Required" 
```

Comment:
Actually since in smarty, do following instead:
``````
<td align="right">{xl t="No Auth# Req"}:</td><td><input type="radio" name="not_req" value="Yes" id="noAuthReq" title="{xl t="Select when No Authorization Number Required"}"